spec/features/manage_deposit_types_link_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.feature 'Manage Deposit Types link in dashboard sidebar', :clean do
+  context 'a logged in user' do
+    let(:user_attributes) do
+      { email: 'test@example.com' }
+    end
+    let(:user) do
+      User.new(user_attributes) { |u| u.save(validate: false) }
+    end
+
+    before do
+      AdminSet.find_or_create_default_admin_set_id
+      login_as user
+    end
+
+    scenario do
+      visit '/dashboard'
+      within '.sidebar' do
+        expect(page).to have_content "Manage self-deposit types"
```

Comment:
Yeah, let's go ahead and move this forward without the tests. I think they are just a bit fragile (changing the labels would be breaking) and aren't really testing anything that could be accidentally removed.

Maybe a test to touch the sidebar as a whole would be appropriate, but short of coming up with a more comprehensive strategy this looks about as good as we can do.